app/views/albums/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-lg-12">
+    <h1><%= @album.title %>
+        <small>Recorded by: <%= @album.artist %></small>
+    </h1>
+    <h4>Ranked: <%= @album.ranking %></h4>
+    <p><%= @album.description %></p>
+    <div>
+      <%= button_to "Upvote", album_upvote_path(@album.id), method: :patch, class: "btn btn-primary" %>
+    </div>
+    <div>
+      <%= link_to "Edit #{@album.title}", edit_album_path(@album.id), class: "btn btn-default" %>
+      <%= link_to "Delete",  { :controller=>"albums", action: :destroy, :id => @album.id}, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger" %>
```

Comment:
We should use the route helper method for this link rather than a hash specifying the route "components": `album_path(@album.id)`.

That route helper is "technically" for the show action, but the path of the route (`/albums/:id`) is the same for both show and delete. By keeping the `method: :delete` part, we generate HTML that sends a DELETE request, so it ends up working and calls "albums#destroy".